orgs/orgs.yml

```diff
@@ -196,6 +196,10 @@ orgs:
         has_projects: false
         has_wiki: false
         private: true
+      bosh-blobstore-clis:
+        description: CLIs for interacting with blobstores
+        has_projects: false
+        has_wiki: false
```

Comment:
The [RFC](https://github.com/cloudfoundry/community/blob/8fc25e9393de1624c84c77f71b2786134b35c852/toc/rfc/rfc-draft-cc-blobstore-storage-cli.md?plain=1#L39-L48) is talking about `storage-cli` repository. I think it make sense to keep it more generic without bosh in the name and storage instead of blobstore. If we want to have it plural like `storage-clis` we should provide that feedback in the RFC and try to agree on that in the RFC. I don't have strong opinion on this.